### PR TITLE
feat: #370 Add import collections

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

**Review by Howl404 — APPROVED** [review]

File: mw-migrations/src/backup/clearAllCollections.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+  const clearAllCollectionsEndTime = new Date();
+  const clearAllCollectionsTime = clearAllCollectionsEndTime.getTime() - clearAllCollectionsStartTime.getTime();
+
+  log(`
+  Start time: ${clearAllCollectionsStartTime}
+  End time: ${clearAllCollectionsEndTime}
+  Total time: ${clearAllCollectionsTime} ms
+
+  Total Documents to deleted: ${totalDocumentsAmount}
```
Comment (tritonJS826) [inline mw-migrations/src/backup/clearAllCollections.ts:60]: you don't show how much read operation executed

File: mw-migrations/src/backup/importAllCollections.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    const importEndTime = new Date();
+    const fullImportTime = importEndTime.getTime() - importStartTime.getTime();
+    const totalModelsImport = dayReportsAmount + goalsAmount + usersAmount + waysAmount + goalMetricsAmount;
+
+    log(`
+    Start time: ${importStartTime}
+    End time: ${importEndTime}
+    Total time: ${fullImportTime} ms
+
+    Total Models to changed: ${totalModelsImport}
```
Comment (tritonJS826) [inline mw-migrations/src/backup/importAllCollections.ts:58]:
```suggestion
    Total Models written: ${totalModelsImport}
```

**Review by tritonJS826 — APPROVED** [review]

File: mw-migrations/src/backup/clearAllCollections.ts (hunk 1 above)
Comment (Ekaterina1994) [reply mw-migrations/src/backup/clearAllCollections.ts:67]: ADdded info about READ operations

**Review by SergioAJS — APPROVED** [review]

File: mw-migrations/src/service/WayService.ts [hunk 3]
```diff
@@ -21,4 +22,45 @@ export class WayService {
...
+
+    await setDoc(docRef, wayDTO);
+      
+    return wayDTO;
+  }
+
+  /**
+   * For import purposes
+   * 
+   * {"seconds":1702731520,"nanoseconds":776000000}
```
Comment (Ekaterina1994) [inline mw-migrations/src/service/WayService.ts:39]: Delete comment

File: mw-migrations/backups/README.md [hunk 4]
```diff
@@ -1,3 +1,5 @@
+2024-01-12T17-47-57.380Z
+
```
Comment (tritonJS826) [inline mw-migrations/backups/README.md:2]:
```suggestion
```

**Review by tritonJS826 — APPROVED** [review]

**Pull request merged**